packages/google/google_configure.js

```diff
@@ -1,6 +1,13 @@
 Template.configureLoginServiceDialogForGoogle.helpers({
   siteUrl: function () {
     return Meteor.absoluteUrl();
+  },
+  siteUrlNoTrail: function () {
+    if (Meteor.absoluteUrl.slice(-1) === '/') {
+      return Meteor.absoluteUrl().slice(0, 1_;
```

Comment:
That underscore is definitely a typo. Also the above line calls `absoluteUrl` without parentheses. I don't think this code has been tested.